components/guided_experience.js

```diff
@@ -37,12 +39,15 @@ const nextButton = css`
 `;
 
 const title = css`
-  font-size: 1.5em !important;
-  color: black;
+  font-size: 14px !important;
+  line-height: 21px;
+  width: auto;
 `;
 
 const jumpButton = css`
-  font-size: 1.5em;
+  font-size: 14px !important;
+  line-height: 21px;
+  color: #006cc9;
```

Comment:
you can use `theme.colour.cerulean` for this colour

Reply:
ok, so all the colors in zeplin are added to the globalTheme?

Reply:
yeah, we added that yesterday I think?

Reply:
@sastels should I replace all the color ref's to hex vs globalTheme within the file?
eg: background-color: #39824d !important;   used on the next button

Reply:
yeah, any ones you find in `theme.js`. `#39824d` is `fernGreen` :)